global-files/.github/labels.yml

```diff
@@ -1,25 +1,25 @@
-# This is an auto-generated file. Please don't edit this file directly.
+# --------------------------------------------------------------- #
+# This is an auto-generated file - Do not manually edit this file #
+# --------------------------------------------------------------- #
 
-# This file is automatically generated by merging two files:
-# 1. The Exercism-wide labels: defined in https://github.com/exercism/org-wide-files/blob/main/synced-files/.github/labels.yml)
-# 2. The repository-specific labels: defined in a `.appends/.github/labels.yml` file (optional)
+# This file is automatically generated by concatenating two files:
 #
-# If you want to use additional labels besides the Exercism-wide labels,
-# don't edit this file directly but instead create a `.appends/.github/labels.yml`
-# file within this repo.
+# 1. The Exercism-wide labels: defined in https://github.com/exercism/org-wide-files/blob/main/global-files/.github/labels.yml
```

Comment:
```suggestion
# 1. The Exercism-wide labels (defined [here](https://github.com/exercism/org-wide-files/blob/main/global-files/.github/labels.yml))
```

I'm not sure if `:` is the right punctuation here. Just a nitpick though

Reply:
🤷